composer.json

```diff
@@ -26,6 +26,7 @@
         "drupal/devel": "dev-8.x-1.x",
         "drupal/drupal-extension": "dev-assert-not-button as 3.1.6",
         "drupal/phing-drush-task": "1.0",
+        "drupal/search_api": "8.1.0-alpha12",
```

Comment:
I can give some more background information. This is used by our continuous integration setup. ContinuousPHP only installs the `require-dev` packages for running the tests. When deploying they are running Composer with the `--no-dev` flag which omits `require-dev` for production builds. We are using these to deploy to the acceptance environments on Amazon. So this means everything which is in `require-dev` will not be there for doing UAT.